drivers/at86rf2xx/at86rf2xx_netdev.c

```diff
@@ -561,6 +568,10 @@ static void _isr(netdev_t *netdev)
                 at86rf2xx_set_state(dev, dev->idle_state);
                 DEBUG("[at86rf2xx] return to state 0x%x\n", dev->idle_state);
             }
+#if defined(MODULE_AT86RF232) || defined(MODULE_AT86RF233)
```

Comment:
Optionally you could change these checks to ifdef AT86RF2XX_XAH_CTRL_2__ARET_FRAME_RETRIES_OFFSET
and only have the model specific checks inside the register header. This way it will be easy to add support for a future at86rf234 or whatever

Reply:
That sounds like a good idea